src/appleseed.shaders/src/appleseed/as_subsurface.osl

```diff
@@ -92,15 +92,33 @@ shader as_subsurface
         string label = "Depth Scale",
         string page = "Subsurface"
     ]],
-    int in_sss_maximum_ray_depth = 2
+    float in_volume_anisotropy = 0.0
+    [[
+        string as_maya_attribute_name = "volumeAnisotropy",
+        string as_maya_attribute_short_name = "sva",
+        string widget = "slider",
+        float min = -1.0,
+        float max = 1.0,
+        int as_maya_attribute_connectable = 0,
+        int as_maya_attribute_keyable = 0,
+        int as_maya_attribute_hidden = 1,
+        int as_blender_input_socket = 0,
+        int gafferNoduleLayoutVisible = 0,
+        string label = "Volume Anisotropy",
+        string page = "Subsurface",
+        string help = "Scattering anisotropy, with negative values producing back scattering, 0 isotropic scattering, and positive values producing forward scattering",
+        int divider = 1
+    ]],
+    int in_sss_maximum_ray_depth = 100
```

Comment:
I'm not sure we want to change this default value, especially since there is no other global control for this in appleseed yet (unlike max diffuse/glossy/specular bounces). I would suggest to leave it at 2.

Reply:
This affects the specular term as well and has a considerable impact on the *glassy* random walk - there, a default of 2 is just too low. 8?

Reply:
Done